tensorflow/compiler/mlir/lite/ir/tfl_op_enums.td

```diff
@@ -59,10 +59,12 @@ def TFL_AFEnum_Relu1 : I32EnumAttrCase<"RELU_N1_TO_1", 2>;
 def TFL_AFEnum_Relu6 : I32EnumAttrCase<"RELU6", 3>;
 def TFL_AFEnum_Tanh  : I32EnumAttrCase<"TANH", 4>;
 def TFL_AFEnum_Sign  : I32EnumAttrCase<"SIGN_BIT", 5>;
+def TFL_AFEnum_Relu0To1 : I32EnumAttrCase<"RELU_0_TO_1", 6>;
```

Comment:
Remove the enum attr work. Tensorflow Lite does not define a value for Relu0To1 so we should not include it unless it is officially added.

Reply:
Yup!

Reply:
Whoops..seems I can't remove this one and got en compilation error in our internal build: Variable not defined: `TFL_AFEnum_Relu0To1`

Reply:
Did you remove the references below as well? As this enum is not defined in TFLite's runtime it would be incorrect to have this enum value set.

Reply:
My bad..I will push those tiny fixes shortly.